app/static/src/scss/base.scss

```diff
@@ -0,0 +1,13 @@
+.banner__container {
```

Comment:
are these names following the BEM naming convention?

Reply:
I believe this is ok. BEM allows for a block__element with no modifier